src/tools/tidy/src/style.rs

```diff
@@ -107,6 +107,12 @@ pub fn check(path: &Path, bad: &mut bool) {
 
         contents.truncate(0);
         t!(t!(File::open(file), file).read_to_string(&mut contents));
+
+        if contents.is_empty() {
+            tidy_error!(bad, "{}: empty file", file.display());
+            return
```

Comment:
I figured the early-return would be fine since the other checks are based on `contents` containing things, but I guess there isn't much harm in going through them anyways.